project/Build.scala

```diff
@@ -1036,7 +1062,7 @@ object Build extends sbt.Build {
   lazy val partest: Project = Project(
       id = "partest",
       base = file("partest"),
-      settings = commonSettings ++ Seq(
+      settings = commonSettings ++ fatalWarningsSettings ++ Seq(
```

Comment:
I am considering to turn of analysis for the ir and the tools. IIUC it does not work anyways: We have default cases everywhere (so exhaustiveness is useless) and we fail reachability everywhere. WDYT?